### PR TITLE
Fix pricenode/docker

### LINKED ISSUE #252: Fix pricenode/docker
The dockerfile is outdated. Needs to be fixed and updated for Haveno.

### PR DESCRIPTION
**Pull request by devhyper**
Resolves #252 
Updates the pricenode docker for haveno.

### CODE REVIEW

**Comment by erciccione** [comment]
Thanks @devhyper! This is very appreciated. I will review this PR in the next days :)

File: pricenode/docker/README.md [hunk 1]
```diff
@@ -21,23 +7,14 @@ Needed software to start a pricenode
...
 See if it's running: `docker ps`
 
-Check the logs: `docker-compose logs`
-
-
-Notes when using CoreOs
-==
-
-Using CoreOs as host OS is entirely optional!
+Check the logs: `docker compose logs`
```
Comment (erciccione) [inline pricenode/docker/README.md:18]: `docker-compose logs` was correct

File: pricenode/docker/README.md [hunk 2]
```diff
@@ -21,23 +7,14 @@ Needed software to start a pricenode
...
 
-* the cloudconfig.yml file is a configuration file for starting a coreos machine
-from scratch.
-* when installing a Coreos server, docker-compose needs to be additionally installed next to the
-already provided docker installation
+Check the tor hostname: `docker exec docker-pricenode-1 cat /var/lib/tor/pricenode/hostname`
```
Comment (erciccione) [inline pricenode/docker/README.md:20]:
```suggestion
Check the tor hostname: `docker exec docker_pricenode_1 cat /var/lib/tor/pricenode/hostname`
```

**Review by erciccione — CHANGES_REQUESTED** [review]
I found only two minor issues, but I'm thinking would probably be good to link to the main 'pricenode' readme from here, so that people know where to find the commands to test their newly launched pricenode.

**Review by erciccione — APPROVED** [review]
Looks good! Thanks :)

**Comment by woodser** [comment]
Thank you!

**Pull request merged**